tests/test_e2e_41_kytos_auth.py

```diff
@@ -0,0 +1,203 @@
+import os
+import time
+import shutil
+import requests
+from tests.helpers import NetworkTest
+from kytos.core.auth import UserController
+
+CONTROLLER = '127.0.0.1'
+KYTOS_API = 'http://%s:8181/api/kytos' % CONTROLLER
+
+
+class TestE2EKytosAuth:
+    net = None
+
+    def setup_method(self, method):
+        """
+        It is called at the beginning of every class method execution
+        """
+        self.net.start_controller(clean_config=True, enable_all=False)
+        self.user_data = {
+            "username": os.environ.get("USER_USERNAME"),
```

Comment:
@Alopalao, I was about to approve and merge this PR, but I've just realized that my initial suggestion of also using env vars here would complicate to run it on GitLab CI that AmLight also has (so we'd need more PRs for little benefit), initially I was thinking of also reusing this testing user, but since no other NApps are using authentication yet, let's remove the env vars and hardcode the values on this `self.user_data` dict, so could you do it? Thanks. Later on, we can use a `pytest` `fixture` to reused when we need to have an existing users for other tests. 

Btw, I've also just run locally the tests as well:

```
+ python3 -m pytest tests/ --timeout=300 -k e2e_41_kytos_auth
============================= test session starts ==============================
platform linux -- Python 3.9.2, pytest-6.2.5, py-1.11.0, pluggy-1.0.0
rootdir: /tests
plugins: timeout-2.0.2
timeout: 300.0s
timeout method: signal
timeout func_only: False
collected 208 items / 200 deselected / 8 selected

tests/test_e2e_41_kytos_auth.py ........                                 [100%]

========== 8 passed, 200 deselected, 35 warnings in 128.41s (0:02:08) ==========
+ tail -f

```

Reply:
Ok, changed.

Reply:
Thanks @Alopalao